test/check-machines-consoles

```diff
@@ -127,7 +125,8 @@ class TestMachinesConsoles(machineslib.VirtualMachinesCase):
         b.wait_text(f"#{name}-terminal", "Disconnected from serial console. Click the connect button.")
 
         b.click(f"#{name}-serialconsole-connect")
-        b.wait_in_text(f"#{name}-terminal .xterm-accessibility-tree > div:nth-child(1)", "Connected to domain")
+        b.wait_in_text(f"#{name}-terminal .xterm-accessibility-tree > div:nth-child(1)",
+                       f"Connected to domain '{name}'")
```

Comment:
So, did I get that right? That'll scroll off as well, but we have a higher chance of catching it because we start polling for this immediately after clicking the connect button. As opposed to the place above where the VM was already running for some indeterminate time?